app/views/CapitalGainsView.scala.html

```diff
@@ -35,7 +37,8 @@
     messages: Messages,
     formPartialRetriever: FormPartialRetriever,
```

Comment:
formPartialRetriever can be deleted from most of the views now that its injectable